sql/core/src/test/scala/org/apache/spark/sql/sources/v2/DataSourceV2SQLSuite.scala

```diff
@@ -38,8 +37,7 @@ class DataSourceV2SQLSuite extends QueryTest with SharedSparkSession with Before
 
   import org.apache.spark.sql.catalog.v2.CatalogV2Implicits._
 
-  private val orc2 = classOf[OrcDataSourceV2].getName
-  private val v2Source = classOf[FakeV2Provider].getName
+  private val v2Format = classOf[FakeV2Provider].getName
```

Comment:
Why rename `v2Source` to `v2Format`? That introduces several unnecessary changes.